tools/pipelines/templates/build-npm-package.yml

```diff
@@ -345,6 +353,13 @@ stages:
               ArtifactName: 'test-files'
               publishLocation: 'Container'
 
+          - task: PublishBuildArtifacts@1
+            displayName: Publish Artifact - Tinylicious Log
```

Comment:
I could envision this new artifact eventually containing other logs as well. We can change the display name at that time though, just pointing it out. I do like that the temp path is `/log` and the artifact name is `log`. If anything I'd use `logs` for both, but just a nit.

Reply:
Hey Alex, that is a good point and I have changed the name of directory to `/logs`. Currently for the artifact file is just the `tinylicious.log` (with a new location), I suppose we may include other logs in the future, but they will be different artifacts. So I just changed the artifact name `log` to `tinyliciousLog`. May I know what do you think?